packages/utils/src/date-utils/checkIsNextDay.ts

```diff
@@ -1,8 +1,10 @@
+import type { OntimeEvent } from 'ontime-types';
+
 import { dayInMs } from './conversionUtils.js';
 
 /**
  * Utility function checks whether a given event is the day after from its predecessor
- * We consider an event to be the day after, if it begins before the start of the previous
+ * We consider an event to be the day after, if it begins on a new day
```

Comment:
is there a reason for the change? this is not the existing logic or what the examples below indicate

Reply:
we now rely on the day offset to tell us this so `before the start of the previous` is no longer exactly what we are testing with

Reply:
in that case, i suggest we remove this sentence

> * We consider an event to be the day after, if it begins on a new day

i fear it doesnt really add anything other than potential for confusion
The examples are also incorrect since we are no longer inferring day after from the times, but from the `dayOffset` fields